controllers/activemqartemis_reconciler.go

```diff
@@ -1225,7 +1289,19 @@ func generateAcceptorSSLOptionalArguments(acceptor brokerv1beta1.AcceptorType) s
 	return sslOptionalArguments
 }
 
-func generateConnectorSSLOptionalArguments(connector brokerv1beta1.ConnectorType) string {
+func (r *ActiveMQArtemisReconcilerImpl) addCreatePemConfigFileCmds(ss *appsv1.StatefulSet, configs []string) {
+	if _, ok := r.PemCfgMap[configs[0]]; !ok {
+		amqName := environments.Retrieve(ss.Spec.Template.Spec.InitContainers, "AMQ_NAME")
+		cfgFileLoc := brokerConfigRoot + "/" + amqName.Value + "/etc/" + configs[0]
+		cmd := " && echo \"source.key=" + configs[1] + "\" > " + cfgFileLoc
+		ss.Spec.Template.Spec.InitContainers[0].Args[1] += cmd
+		cmd = " && echo \"source.cert=" + configs[2] + "\" >> " + cfgFileLoc
+		ss.Spec.Template.Spec.InitContainers[0].Args[1] += cmd
+		r.PemCfgMap[configs[0]] = true
+	}
+}
```

Comment:
The operator could add the pemcfg file to configMap or secret, long command line could cause issues and the paths could include special chars to escape.

Reply:
+1